### DISCUSSION TITLE
Fonts for RiMusic

### DISCUSSION THREAD

**Discussion by ikanakova** [fast4x/RiMusic#349] [Ideas]
Discussion for suggestions of optional fonts for RiMusic.

**Rules:**
- Multilanguage fonts only - must support all languages into which RiMusic is [translated](https://crowdin.com/project/rimusic).

- Must be under the Open Font License (OFL) or other open license.

- Always provide a link to the font's website, which must include at least the license and glyphs.

I would like to ask all translators to check that the suggested fonts support all characters in their language.

**Comment by fast4x** [comment]
How about a third font?

**Comment by fast4x** [comment]
https://fonts.google.com/noto

**Comment by fast4x** [comment]
Adding Noto Sans font Family

**Comment by fast4x** [comment]
Unfortunately adding the font Noto, the apk is enlarged over 6MB, I removed it.

**Comment by fast4x** [comment]
What think about this fonts? #2283

**Reply by ikanakova** [reply]
I like Roboto and DM Sans (and not only because they are open source 😆) I don't like fonts like Okesip, at least not in applications. Rustic Story is better than Okesip, but worse than Roboto and DM Sans.
